### PR TITLE
Listen to the user status event to update the status immediately

### LINKED ISSUE #4132: Own status indicator only updates after reload
I just tested the new online status feature on c.nc.com... really nice!

I noticed that when I change my own status I need to reload the page to update the indicator for myself in the participant list. Even changing rooms doesn't update it. That's quite irritation because as a user you get confused about whether your status change was successful or not.

### PR DESCRIPTION
**Pull request by nickvergessen**
### Steps
1. Go to any page that uses the Avatar component to show your own avatar, e.g. a Talk conversation participant tab
2. Change your status
3. Wait until your avatar shows the new status :skull_and_crossbones: 

With this PR the text + emoji-icon will update. The dot-icon on the avatar will work after https://github.com/nextcloud/nextcloud-vue/pull/1405

Relevant event is emitted in server:
https://github.com/nextcloud/server/pull/22801

Fix #4132

### CODE REVIEW

File: src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue [hunk 1]
```diff
@@ -75,7 +76,28 @@ export default {
 		},
 	},
 
+	mounted() {
+		subscribe('user_status:status.updated', this.userStatusUpdated)
```
Comment (skjnldsv) [inline src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue:80]: didn't we said `user_status:status:updated` ?
```suggestion
		subscribe('user_status:status:updated', this.userStatusUpdated)
```

Comment (skjnldsv) [reply src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue:80]: https://github.com/nextcloud/server/blob/659bea6de2fea0656ea982dd0109ec259223ab61/core/src/views/UnifiedSearch.vue#L352

Comment (skjnldsv) [reply src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue:80]: https://github.com/nextcloud/server/pull/22526#discussion_r480920065
Maaaan, I'm so confused, let's just pick one and move on xD

Comment (skjnldsv) [reply src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue:80]: I'll move the unified search one to a dot too :sweat_smile: 
cc @raimund-schluessler

Comment (skjnldsv) [reply src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue:80]: (tomorrow, I'm too sleepy now :zzz: )

Comment (raimund-schluessler) [reply src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue:80]: Maybe we should write that down somewhere. Changing it to a dot requires alls apps to also adjust again (I don't know how many use this event already, though).

Comment (nickvergessen) [reply src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue:80]: https://github.com/nextcloud/nextcloud-event-bus/issues/131#issuecomment-683696800 also had a dot in the example. I mean we can also subscribe to both or something. But anyway it needs fixing in the server then I guess.

Comment (skjnldsv) [reply src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue:80]: > Maybe we should write that down somewhere. Changing it to a dot requires alls apps to also adjust again (I don't know how many use this event already, though).

not much, only you I think :)
Let's go for the dot, whatever was discussed and I misunderstood seems to not be what @ChristophWurst suggested in https://github.com/nextcloud/nextcloud-event-bus/issues/131#issuecomment-683660316

Comment (skjnldsv) [reply src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue:80]: Also updated main post https://github.com/nextcloud/nextcloud-event-bus/issues/131#issue-658951244

Comment (raimund-schluessler) [reply src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue:80]: Fine with me, let's move to a dot then.

Comment (skjnldsv) [reply src/components/RightSidebar/Participants/CurrentParticipants/CurrentParticipants.vue:80]: Please see https://github.com/nextcloud/server/pull/22868 @nickvergessen @raimund-schluessler

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**